dask/array/slicing.py

```diff
@@ -445,14 +481,9 @@ def issorted(seq):
     >>> issorted([3, 1, 2])
     False
     """
-    if not seq:
+    if len(seq) == 0:
         return True
-    x = seq[0]
-    for elem in seq[1:]:
-        if elem < x:
-            return False
-        x = elem
-    return True
+    return np.all(sorted(seq) == seq)
```

Comment:
If you want a vectorized implementation, perhaps try:
```python
>>> def f1(seq): return np.all(sorted(seq) == seq)
>>> def f2(seq):
...:    seq = np.asanyarray(seq)
...:    return np.all(seq[:-1] <= seq[1:])
...:
>>> seq = np.random.randint(0, 100, 10000)
>>> seq
array([46, 85, 75, ...,  8, 65, 63])
>>> len(seq)
10000
>>> 
>>> %timeit f1(seq)
100 loops, best of 3: 4.17 ms per loop
>>> %timeit f2(seq)
The slowest run took 6.39 times longer than the fastest. This could mean that an intermediate result is being cached.
10000 loops, best of 3: 16.3 µs per loop
```